medusa/app.py

```diff
@@ -271,7 +271,7 @@
 TV_DOWNLOAD_DIR = None
 UNPACK = False
 SKIP_REMOVED_FILES = False
-ALLOWED_EXTENSIONS = "srt,nfo,sub,idx"
+ALLOWED_EXTENSIONS = ('srt', 'nfo', 'sub', 'idx')
```

Comment:
I need to check why I initiated this as a tuple. Think it should be list though.